{{cookiecutter.project_slug}}/src/redux/mainSaga.js

```diff
@@ -11,8 +11,8 @@ export function* mainSaga() {
   yield all([
     takeEvery("TEST/ALO", helloSaga),
     // other sagas go here
-    {% if cookiecutter.has_email_auth_blueprint == "y" %}EmailAuthSaga,{% endif %}
-    {% if cookiecutter.has_calendar_blueprint == "y" %}CalendarSaga,{% endif %}
```

Comment:
Before we merge this PR this needs to be implemented in the `install.sh` of the `EmailAuth` and `Calendar` screens repositories.